_posts/2022-10-24-wildfly-jakartaee.adoc

```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title:  "Introduction to working on/with Jakarta EE Specifications for WildFly users"
+date:   2022-10-24
```

Comment:
@scottmarlow Please update the file name and this value to today or some future date. Future date would be if you want this post separated a bit from the other posts today.

Reply:
Also, please ping me in chat when updated; i'll merge it right away. Otherwise it might get lost in the task pile. ;)